### PR TITLE
ActorContext Flags

### PR DESCRIPTION
**Pull request by engineer124**
This looks at the two types flags involved in `ActorContext`.

What was previously `actorCtx.flags` is proposed to be given the more specific `actorCtx.sceneFlags` since it is specifically scene flags that it stores, and the more general `actorCtx.unk5` is proposed to be given the more general `actorCtx.flags` name, along with defines for it.

### CODE REVIEW

**Review by hensldm — APPROVED** [review]
Seems like a good change to me :)

**Review by EllipticEllipsis — APPROVED** [review]
Seems okay to me. We'd normally name these 1,2,4,8,... but since they're so little-used this is probably fine.

File: tools/namefixer.py [hunk 1]
```diff
@@ -698,6 +698,8 @@
     "globalCtx->envCtx.unk_DC": "play->envCtx.lightBlend",
     "globalCtx->interfaceCtx.unk_21E": "play->interfaceCtx.bButtonDoAction",
 
+    "play->actorCtx.flags": "play->actorCtx.sceneFlags",
+    "play->actorCtx.unk5": "play->actorCtx.flags",
```
Comment (AngheloAlf) [inline tools/namefixer.py:702]: This produces a recursive problem... Running `./tools/namefixer.py` should warn about it

**Review by AngheloAlf — CHANGES_REQUESTED** [review]

File: tools/namefixer.py (hunk 1 above)
Comment (engineer124) [reply tools/namefixer.py:702]: I wasn't too sure about this, should I just leave this out then?

Comment (AngheloAlf) [reply tools/namefixer.py:702]: I think you could leave them commented out

**Review by AngheloAlf — APPROVED** [review]

**Pull request merged**